port_ocean/core/defaults/initialize.py

```diff
@@ -130,10 +130,8 @@ async def _initialize_defaults(
         )
         await asyncio.gather(
             *(
-                port_client.delete_blueprint(
-                    blueprint["identifier"], should_raise=False
-                )
-                for blueprint in defaults.blueprints
+                port_client.delete_blueprint(identifier, should_raise=False)
+                for identifier in e.blueprints_to_rollback
```

Comment:
lets move those to a different pr so ull be able to ship this integration with them in the first version without releasing a new one